### PR TITLE
Patryk/cross entropy balance

### PR DESCRIPTION
**Pull request by pkrukowski1**
I added an option to use a dynamic factor multiplied by the current task loss.

### CODE REVIEW

File: src/method/ewc.py [hunk 1]
```diff
@@ -98,7 +98,9 @@ def forward(self, x, y, loss, preds):
         self.data_buffer.add((x, y))
 
         if self.task_id > 0:
-            loss += self.alpha*param_change_loss(self.module, self.fisher_diag, self.params_buffer, self.head_opt)
+            loss = self.alpha*param_change_loss(self.module, self.fisher_diag, self.params_buffer, self.head_opt)
+        else:
+            loss = torch.tensor(0.0)
```
Comment (OneAndZero24) [inline src/method/ewc.py:103]: This seems odd, for `task_id == 0` loss will be 0 -> no optimization. Also please keep old version of EWC with `+=` for running without dynamic loss scalling

**Review by OneAndZero24 — CHANGES_REQUESTED** [review]
Everything ok, apart from EWC.

**Review by OneAndZero24 — APPROVED** [review]
👍
